azure-arc-for-servers/README.md

```diff
@@ -0,0 +1,47 @@
+After connect your machine to Azure (`called connected machine` here), 
+you can perform management tasks as described in the [Azure ARC document][scenario] including assigning Azure policies for security compliance, monitoring your machine using Log Analysis, tagging your machines etc. 
+Apart from that, you may want to develop an application that runs on the connected machines but needs to connect to your Azure services. As we know, one of the challenges while building cloud applications is to manage the credentials because they cannot be checked-in as a part of your code or stored in your dev box. Fortunately, Azure ARC for Servers makes it easy for you. 
+
+## Acquiring Azure Access Token
+On the connected machine, you can request Azure Hybrid Instance Metadata Service (himds) for Azure access token. 
+Once your app gets a token, it can be connected to your Azure services by passing the token in the Authorization header using the Bearer scheme as if from an Azure VM.
+For example, assuming your app needs to connect to your Azure Cosmos DB, first you need to retrieve the connection string and other secrets stored in Azure Key Vault. The [sample code][kv] illustrates how to retrieve Azure access token from on ARC servers. 
+Thanks to [PowerShell 7 Preview 6 release][ps6], we can now use "Skip Error Check for Web Cmdlets" feature as shown in the example.
+
+When you run the sample code, you may get the AccessDenied error. 
+This is because after you connect your machine to Azure, there is an identity created representing your machine, but you have not granted any permissions to your machine yet. 
+Therefore, even though you successfully retrieve the token, this token can do nothing.
+
+## Grant Permissions to Connected Machines
+Next you need to grant your connected machine for a read permission to access to your Key Vault using Azure RBAC. 
+You can manage this access control through Azure portal, Azure CloudShell, or run the [sample code][permission] on your laptop, desktop or any of your machines. 
+Please note changing RBAC roles require you to be the owner or administer of your subscription.
+
+## Retrieve Secrets from Key Vault
+Now you have completed the role assignment. You should be able to get the secrets stored in you Key Vault. Once you have your secrets, for example, Cosmos Connection string, your app can start performing your business logic. 
+
+
+Other than getting Azure access token, you can find out metadata information including subscription, resouceGroup etc where your  machine connected to using the REST api just like on Azure VMs, for example,
+
+```
+Invoke-WebRequest -Uri http://localhost:40342/metadata/instance?api-version=2019-11-01 -Headers @{Metadata="True"} -UseBasicParsing
+
+```
+
+Please give it a try and welcome your feedback, bugs and feature requests for Azure ARC for Servers!
+  
+
+[Ignite]:https://youtu.be/jnUiJi4hts4?t=869
+[overview]:https://docs.microsoft.com/en-us/azure/azure-arc/servers/overview
```

Comment:
Remove the locale "en-us" from all URLs on microsoft.com.

Reply:
ok.